stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -67,15 +67,10 @@ class VhloToVersionConverter : public TypeConverter {
   }
 };
 
-FailureOr<Version> parseTargetVersion(llvm::StringRef versionRef) {
-  if (versionRef == "current") return Version::getCurrentVersion();
-  return Version::fromString(versionRef);
-}
```

Comment:
If we're cleaning up version alias parsing, we could also clean up similar logic in getMinVersion / getMaxVersion methods defined in TableGen for VHLO program elements.